app/controllers/admin/routes.py

```diff
@@ -141,8 +140,12 @@ def editEvent(eventId):
     userHasRSVPed = EventRsvp.get_or_none(EventRsvp.user == g.current_user, EventRsvp.event == event)
     isPastEvent = (datetime.now() >= datetime.combine(event.startDate, event.timeStart))
     program = event.singleProgram
-
     isProgramManager = hasPrivilege(g.current_user,program)
+
+    if (g.current_user.isStudent or g.current_user.isFaculty) and not isProgramManager:  # only formats to 12 hour time if user doesn't have access (for display purposes)
+        eventData['timeStart'] = format24to12HourTime(eventData['timeStart'])
+        eventData['timeEnd'] = format24to12HourTime(eventData['timeEnd'])
```

Comment:
Doesn't the time picker require 24 hour time? I thought that was the whole issue you dealt with in the other timepicker PR

Reply:
Yes the time picker does, not the text for people w/ out edit access